libs/pages/login/src/lib/feature/container/container.tsx

```diff
@@ -1,15 +1,16 @@
 import { useEffect } from 'react'
 import { useNavigate } from 'react-router'
-import { LayoutLogin, Login } from '@console/pages/login/ui'
 import { useDocumentTitle } from '@console/shared/utils'
 import { OVERVIEW_URL } from '@console/shared/router'
 import { AuthEnum, useAuth } from '@console/shared/auth'
 import { useOrganization } from '@console/domains/organization'
 import { useProjects } from '@console/domains/projects'
+import LayoutLogin from '../../ui/layout-login/layout-login'
+import Login from '../../ui/login/login'
 
 // import posthog from 'posthog-js'
 
-export function LoginPage() {
+export function Container() {
```

Comment:
Why container and not `PageLoginFeature`? For each Page we gonna have its Container if I understand well?

Reply:
I agree, better to have `PageLoginFeature` I rename this one